aws/cluster/variables.tf

```diff
@@ -75,6 +75,22 @@ variable "kube-dns" {
   }
 }
 
+# kube-proxy
+variable "kube-proxy" {
+  type = object({
+    enabled          = bool
+    clusterCIDR      = string
+    cpuRequest       = string
+    hostnameOverride = string
+  })
+  default = {
+    enabled          = false
```

Comment:
Does the default of false match the current usage?

Reply:
Hmm, so this is indirectly affected by `var.container-networking`? Might be worth reflecting this in the [spec](https://github.com/wandera/karch/pull/57/files#diff-2c55de97b5f7202e959655dfd70fee2a2b07a8a7e00ebee9140933ac4ee332ecR106)? 🤔  Not that important probably :-)